src/webview/markdownEngine.ts

```diff
@@ -107,4 +138,29 @@ class MarkdownEngine {
     }
 }
 
+// tslint:disable-next-line: max-classes-per-file
+class MarkdownConfiguration {
+
+    public readonly extRoot: string; // root path of vscode built-in markdown extension
+    public readonly lineHeight: number;
+    public readonly fontSize: number;
+    public readonly fontFamily: string | undefined;
+
+    public constructor() {
+        const markdownConfig: vscode.WorkspaceConfiguration = vscode.workspace.getConfiguration("markdown");
+        this.extRoot = path.join(vscode.env.appRoot, "extensions", "markdown-language-features");
+        this.lineHeight = Math.max(0.6, +markdownConfig.get<number>("preview.lineHeight", NaN));
+        this.fontSize = Math.max(8, +markdownConfig.get<number>("preview.fontSize", NaN));
+        this.fontFamily = this.resolveFontFamily(markdownConfig);
+    }
+
+    private resolveFontFamily(config: vscode.WorkspaceConfiguration): string | undefined {
+        let fontFamily: string | undefined = config.get<string | undefined>("preview.fontFamily", undefined);
```

Comment:
I think just `config.get<string>("preview.fontFamily")` would be fine

Reply:
Any thoughts about this one?

Reply:
Yes, I analysed the code and thought that `undefined` is not needed.

We can just switch to `config.get<string>("preview.fontFamily", "")`.

And when generating css style by config, since `this.config.fontFamily === ""` is falsy, the behavior will still be the same:
```
`    ${this.config.fontFamily ? `font-family: ${this.config.fontFamily};` : ``}`,
```